### PR TITLE
Mail when rent is created

### PR DESCRIPTION
**Pull request by truccomarcos**
# Summary
- Added mail functionality to notify when a rent is created
- Mail format defined
# Trello Card
https://trello.com/c/u37QVH8o/11-mandar-email-al-usuario-cuando-se-crea-un-rent

(Mail tested with letter opener)
screenshot from 2017-06-13 16-04-21

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -39,13 +39,16 @@ gem 'versionist'
 gem 'jwt'
 gem 'faker'
 gem 'active_model_serializers', '~> 0.10.0'
+gem "factory_girl_rails", "~> 4.0"
```
Comment (hdf1986) [inline Gemfile]: Hmm why is this here again? :P

File: Gemfile [hunk 2]
```diff
@@ -62,6 +65,7 @@ group :development do
   gem 'spring-watcher-listen', '~> 2.0.0'
   gem 'rubocop'
   gem 'sass'
+  gem "letter_opener"
```
Comment (hdf1986) [inline Gemfile]: Shouldn't be development & test group instead of just test?

File: app/views/rent_mailer/new_rent_notification.text.slim [hunk 3]
```diff
@@ -0,0 +1,3 @@
+' Rent#new_rent_notification
```
Comment (hdf1986) [inline app/views/rent_mailer/new_rent_notification.text.slim]: If we aren't using this file, please delete it

File: config/environments/development.rb [hunk 4]
```diff
@@ -53,4 +53,16 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.action_mailer.delivery_method = :letter_opener
+  config.action_mailer.smtp_settings = {
+    :authentication => :plain,
+    :address => "smtp.mailgun.org",
```
Comment (hdf1986) [inline config/environments/development.rb]: Can we set this from environment too?

File: spec/fixtures/rent_mailer/new_rent_notification [hunk 5]
```diff
@@ -0,0 +1,3 @@
+RentMailer#new_rent_notification
```
Comment (hdf1986) [inline spec/fixtures/rent_mailer/new_rent_notification]: If we don't use this file just delete it

File: app/workers/rent_mailer_worker.rb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+class RentMailerWorker
+  include Sidekiq::Worker
+
+  def perform(rent_id)
+    @rent = Rent.find(rent_id)
```
Comment (hdf1986) [inline app/workers/rent_mailer_worker.rb]: What happens when the rent was deleted? remember that sidekiq is asynchronous, so can happen a lot of things between you trigger the worker and it really performs the procces, even the rent can be deleted in that lapsus

Comment (truccomarcos) [inline app/workers/rent_mailer_worker.rb]: The worker now checks if rent is null before calling the mailer function
